src/models/core/ClientSdk.ts

```diff
@@ -0,0 +1,21 @@
+
+/**
+ * Additional agents to add to the Client-SDK HTTP header.
+ *
+ * @public
+ **/
+export interface CustomClientSdk {
```

Comment:
I think we should consider renaming this `interface` and the one below. The name is closely tethered to an internal implementation detail (the name of the header we use). But, that's not the level of abstraction a user operates on. They're operate at a higher level of abstraction. They need to know that this is a way for them to add other Answers Agents and have them be passed on LiveAPI calls. I don't think the name `ClientSdk` or `CustomClientSdk` will be very intuitive for them.

Reply:
What do you think of `AnswersAgents` and `CustomAnswersAgents` instead?

Reply:
I actually feel like we should try to be relatively consistent with the API here, since I think these headers end up getting surfaced in the data hub as CLIENT_SDK (or something similar?)

Plus the only people that are going to even use this interface are library maintainers who want to use core in their own libraries, it's not your average joe who just wants to build an answers experience.

Reply:
I agree the name is unintuitive... but I think it's better to just have it be consistent all the way through here (maybe Rose can weigh in more on this though). That being said I don't think our interfaces have to be named exactly `ClientSdk`  or `CustomClientSdk` but they should somewhat resemble the HTTP header?

Reply:
If `CLIENT_SDK` is the key in the data hub, then I would agree with you Oshi. I do think we could improve the docs a bit though.

Reply:
I updated the docs for `CustomClientSdk`. Let me know if more clarification is needed